osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -3244,6 +3219,33 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.svg"
       }
+    },
+    {
+      "description": "The native token of Lambda",
+      "denom_units": [
+        {
+          "denom": "ibc/80825E8F04B12D914ABEADB1F4D39C04755B12C8402F6876EE3168450C0A90BB",
+          "exponent": 0,
+          "aliases": ["ulamb"]
+        },
+        {
+          "denom": "lamb",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/80825E8F04B12D914ABEADB1F4D39C04755B12C8402F6876EE3168450C0A90BB",
+      "name": "Lambda",
+      "display": "lamb",
+      "symbol": "LAMB",
+      "ibc": {
+        "source_channel": "channel-2",
+        "dst_channel": "channel-378",
+        "source_denom": "ulamb"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lambda/images/lambda.png"
+      },
+      "coingecko_id": "lambda"
```

Comment:
It looks likke on CoinGecko this points to the wrong lambda project. Can you confirm?

Reply:
Its the same project, we launched our mainnet and we deploy a bridge to bridge ERC20 Lamb to mainnet.

Here is our bridge frontend   https://app.lambda.im/crosschain/ethereum_bridge

Reply:
Btw, i would like to know how to get marked as `verified`